cypress/support/helper.js

```diff
@@ -41,3 +51,39 @@ export function setSlidebar(selector, value) {
     range.dispatchEvent(new Event('change', { value, bubbles: true }));
   });
 }
+
+function shouldHaveCssElementWidth(size) {
+  return cy.should('have.css', 'width', size);
+}
+
+export function checkTheSizeOfTheElement(element, size) {
+  switch (size) {
+    case dialogSizes.extraSmall.name:
+      shouldHaveCssElementWidth(dialogSizes.extraSmall.width);
+      break;
+    case dialogSizes.small.name:
+      shouldHaveCssElementWidth(dialogSizes.small.width);
+      break;
+    case dialogSizes.mediumSmall.name:
+      shouldHaveCssElementWidth(dialogSizes.mediumSmall.width);
+      break;
+    case dialogSizes.medium.name:
+      shouldHaveCssElementWidth(dialogSizes.medium.width);
+      break;
+    case dialogSizes.mediumLarge.name:
+      shouldHaveCssElementWidth(dialogSizes.mediumLarge.width);
+      break;
+    case dialogSizes.large.name:
+      shouldHaveCssElementWidth(dialogSizes.large.width);
+      break;
+    case dialogSizes.extraLarge.name:
+      shouldHaveCssElementWidth(dialogSizes.extraLarge.width);
+      break;
+    default: throw new Error(`Size: ${size} is not defined`);
+  }
+}
+
+export function clickOntoESCKey() {
```

Comment:
better name is `pressEscKey` instead of click

Reply:
fixed